x-pack/plugins/search_indices/public/components/indices/details_page.tsx

```diff
@@ -248,23 +237,29 @@ export const SearchIndexDetailsPage = () => {
               </EuiFlexGroup>,
             ]}
           />
-          <EuiPageTemplate.Section grow={false}>
+          <EuiPageTemplate.Section
+            grow={false}
+            restrictWidth
+            css={{ padding: `0 ${euiTheme.size.l} ${euiTheme.size.l}` }}
+          >
             <EuiFlexGroup direction="column">
-              <EuiFlexItem>
-                <EuiFlexGroup css={{ overflow: 'auto' }}>
-                  <EuiFlexItem css={{ flexShrink: 0 }}>
-                    <ConnectionDetails />
-                  </EuiFlexItem>
-                  <EuiFlexItem css={{ flexShrink: 0 }}>
-                    <ApiKeyForm />
-                  </EuiFlexItem>
-                </EuiFlexGroup>
-              </EuiFlexItem>
-              <EuiFlexItem>
-                <EuiFlexGroup>
-                  <QuickStats index={index} mappings={mappings} />
-                </EuiFlexGroup>
-              </EuiFlexItem>
+              <EuiFlexGroup direction="column">
+                <EuiFlexItem>
+                  <EuiFlexGroup css={{ overflow: 'auto' }} wrap>
+                    <EuiFlexItem grow={false} style={{ minWidth: 400 }}>
```

Comment:
i would prefer `css` instead of `style`.

Reply:
yes good spot. we should be using css when we can.